cpimauth_settings.php

```diff
@@ -0,0 +1,276 @@
+<?php
+class cpimauth_settings_page
```

Comment:
You're right, that was my mistake. I renamed one of the files and forgot to remove the old one. I will delete this one.

---
In reply to: [71243107](https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect/pull/3#discussion_r71243107) [](ancestors = 71243107,71237235,71235409)